### DISCUSSION TITLE
Proposal For Chair and Vice-Chair Role

### DISCUSSION THREAD

**Discussion by hexsecs** [uptane/uptane.github.io#180] [Ideas]
As we are all aware, Lois did a fantastic job over the past several years organizing and keeping the Uptane community abreast of meeting topics and minutes. With her retirement, there has been large void in many of the things that she was doing. While not explicitly given the title of the role, she did many of the things that a standards group chair would be doing (lead the meetings, publish meeting minutes, etc). With her absence, this role is not being filled, which is needed for a successful standards group.
 
I propose that we formally create a chair and vice chair role within the Uptane group. I’m not necessarily nominating myself for either of these positions, however, the role is very much needed if we want to continue as a standards organization. Additionally a defined role will help whomever takes on that role justify their effort to their organization.
 
**Proposed chair expectations:**
* Lead Uptane biweekly meetings
* Publish meeting agenda prior to meeting
* Publish meeting minutes after the meeting
* Lead consensus building on principal discissions 
* Coordinate backup if they cannot attend a meeting

**Proposed vice-chair expectations:**
* Back up vice-chair in case they cannot attend meeting or complete normal task

**Comment by mnm678** [comment]
As discussed in the 2/13 meeting, we are moving forward with this proposal. An issue for nominating chairs will be opened in the community repository

**Comment by JustinCappos** [comment]
I vote to approve this.

**Comment by tkfu** [comment]
I'll add my yes vote as well. @hexsecs / @mnm678 : who's gonna open the issue in the [community repo](https://github.com/uptane/community/issues)? I can do it if we're ready to go.

**Comment by jhdalek55** [comment]
I'm happy to see this moving forward.

**Comment by tkfu** [comment]
I've opened up the issue [here](https://github.com/uptane/community/issues/3).
